src/contracts/IbexaTestKernel.php

```diff
@@ -0,0 +1,289 @@
+        Connection::class,
+        Repository\Repository::class,
+        Repository\ContentService::class,
+        Repository\ContentTypeService::class,
+        Repository\LanguageService::class,
+        Repository\LocationService::class,
+        Repository\ObjectStateService::class,
+        Repository\PermissionResolver::class,
+        Repository\RoleService::class,
+        Repository\SearchService::class,
+        Repository\SectionService::class,
+        Repository\UserService::class,
+    ];
+
+    /**
+     * @var iterable<string, class-string>
+     */
+    protected const EXPOSED_SERVICES_BY_ID = [];
+
+    /**
+     * @return string a service ID that service aliases will be registered as
+     */
+    public static function getAliasServiceId(string $id): string
+    {
+        return 'test.' . $id;
+    }
+
+    /**
+     * @return iterable<string>
+     */
+    public function getSchemaFiles(): iterable
+    {
+        yield $this->locateResource('@IbexaCoreBundle/Resources/config/storage/legacy/schema.yaml');
+    }
+
+    /**
+     * @return iterable<\Ibexa\Contracts\Core\Test\Persistence\Fixture>
+     */
+    public function getFixtures(): iterable
+    {
+        yield new YamlFixture(__DIR__ . '/Resources/test_data.yaml');
+    }
+
+    public function getCacheDir(): string
+    {
+        return sys_get_temp_dir() . '/ibexa-test-kernel/' . md5(serialize(getenv())) . md5(static::class);
+    }
+
+    public function getBuildDir(): string
+    {
+        return sys_get_temp_dir() . '/ibexa-test-kernel-build/' . md5(serialize(getenv())) . md5(static::class);
+    }
+
+    public function registerBundles(): iterable
+    {
+        yield new SecurityBundle();
+        yield new IbexaCoreBundle();
+        yield new IbexaLegacySearchEngineBundle();
+        yield new JMSTranslationBundle();
+        yield new FOSJsRoutingBundle();
+        yield new FrameworkBundle();
+        yield new LiipImagineBundle();
+        yield new TwigBundle();
+        yield new DoctrineBundle();
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function registerContainerConfiguration(LoaderInterface $loader): void
+    {
+        $loader->load(static function (ContainerBuilder $container): void {
+            $container->setParameter('ibexa.core.test.resource_dir', self::getResourcesPath());
+        });
+
+        $this->loadConfiguration($loader);
+        $this->loadServices($loader);
+
+        $loader->load(static function (ContainerBuilder $container): void {
+            self::prepareIOServices($container);
+            self::createPublicAliasesForServicesUnderTest($container);
+            self::setUpTestLogger($container);
+        });
+    }
+
+    /**
+     * @throws \Exception
+     */
+    protected function loadConfiguration(LoaderInterface $loader): void
+    {
+        $loader->load(self::getResourcesPath() . '/config/doctrine.php');
+        $loader->load(self::getResourcesPath() . '/config/ezpublish.yaml');
```

Comment:
Okay, let's not do this please :D
```suggestion
        $loader->load(self::getResourcesPath() . '/config/ibexa.yaml');
```

Reply:
Fixed in d408409a.